ee/desktop/runner/runner.go

```diff
@@ -410,11 +410,14 @@ func (r *DesktopUsersProcessesRunner) socketPath(uid string) (string, error) {
 func (r *DesktopUsersProcessesRunner) desktopCommand(executablePath, uid, socketPath string) (*exec.Cmd, error) {
 	cmd := exec.Command(executablePath, "desktop")
 
-	cmd.Env = []string{
+	cmd.Env = append(
+		// if os.Environ() is not included, windows wiil not render the icon, producing the error:
+		// unable to write icon data to temp file: open C:\\windows\\systray_temp_icon_...: Access is denied
+		os.Environ(),
```

Comment:
Hrm. I would be inclined to try to figure out _which_ part of the environment matters. And to consider varying behavior by OS. But if we can isolate it, we can just pass what we need